functions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+function bottlesHtml(array $allBottles): string {
```

Comment:
the name of any function should ideally be a verb e.g. createBottleHtml or something, bottlesHtml is more like a name for a variable